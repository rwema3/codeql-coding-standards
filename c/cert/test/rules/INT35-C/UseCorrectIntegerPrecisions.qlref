rules/INT35-C/UseCorrectIntegerPrecisions.ql