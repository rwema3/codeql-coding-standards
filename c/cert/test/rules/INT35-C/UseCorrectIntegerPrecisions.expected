| test.c:11:12:11:31 | sizeof(unsigned int) | sizeof operator used to determine the precision of an integer type. |
| test.c:27:25:27:42 | sizeof(signed int) | sizeof operator used to determine the precision of an integer type. |
